assets/app/styles/_core.less

```diff
@@ -389,6 +389,21 @@ label.checkbox {
   }
 }
 
+.default-padding {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
```

Comment:
Let's remove this and use Ben's new spacing classes, `pad-top-xl` and `pod-bottom-xl`.